lib/creek/shared_strings.rb

```diff
@@ -21,7 +21,7 @@ def parse_shared_strings
         xml = Nokogiri::XML::Document.parse doc
 
         @dictionary ||= xml.css('si').map do |node|
-          options[:with_html] ? node : text_from(node, options)
+          node
```

Comment:
Are you sure you are not breaking any behavior?

Reply:
yes, it is an internal API
`text_from` moved to SharedStringConvert